cmd/s3_del.go

```diff
@@ -14,8 +14,8 @@ var (
 // CliS3CmdDel is the Cobra CLI call
 func CliS3CmdDel() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "del BUCKET/OBJECT",
-		Short: "Delete bucket",
+		Use:   "del BUCKET/FILE",
+		Short: "Delete file from bucket",
```

Comment:
We should test this, does `del` link to `rb` if the bucket is empty?

Reply:
I added the test for del

I didn't checked the behavior in an empty bucket. I would be more logical not deleting the bucket.